include/SDL3/SDL_gpu.h

```diff
@@ -2359,35 +2357,17 @@ extern SDL_DECLSPEC SDL_GPUShaderFormat SDLCALL SDL_GetGPUShaderFormats(SDL_GPUD
  * Driver Branch: promo490_3_Google
  * ```
  *
- * ### `SDL_PROP_GPU_DEVICE_DEBUG_VULKAN_CONFORMANCE_STRING`
- *
- * When using the Vulkan backend, contains the highest Vulkan version number
- * that the system driver claims that the underlying hardware conforms to.
- * This is self-reported and may not be truthful.
- *
- * Strings that have been found in the wild include:
- *
- * - 0.0.0.0
- * - 1.0.0.0
- * - 1.3.8.2
- *
  * \param device a GPU context to query.
- * \returns a valid property ID or 0. A value of 0 indicates that no
- *          properties are available and is not an error. The returned
- *          property group is owned by SDL and has the same lifetime as the
- *          containing context. It should not be destroyed manually when no
- *          longer needed.
+ * \returns a valid property ID on success or 0 on failure; call SDL_GetError() for more information.
  *
  * \since This function is available since SDL 3.4.0.
  */
-extern SDL_DECLSPEC SDL_PropertiesID SDLCALL SDL_GetGPUDeviceDebugProperties(
-    SDL_GPUDevice *device);
+extern SDL_DECLSPEC SDL_PropertiesID SDLCALL SDL_GetGPUDeviceProperties(SDL_GPUDevice *device);
 
 #define SDL_PROP_GPU_DEVICE_DEBUG_NAME_STRING               "SDL.gpu.device.debug.name"
 #define SDL_PROP_GPU_DEVICE_DEBUG_DRIVER_NAME_STRING        "SDL.gpu.device.debug.driver_name"
 #define SDL_PROP_GPU_DEVICE_DEBUG_DRIVER_VERSION_STRING     "SDL.gpu.device.debug.driver_version"
 #define SDL_PROP_GPU_DEVICE_DEBUG_DRIVER_INFO_STRING        "SDL.gpu.device.debug.driver_info"
```

Comment:
Should we rename these too? These aren't really debug properties, they're always available.

Reply:
Yes, probably. I'll update them.